{
 "/api/2.0/accounts/{account_id}/vpc-endpoints/{vpc_endpoint_id}": {
  "delete": {
   "operationId": "vpc-endpoints-delete",
   "externalDocs": {
    "url": "https://docs.databricks.com/api/account/vpcendpoints/delete"
   },
   "x-stackQL-resource": "vpc_endpoints",
   "x-stackQL-method": "delete",
   "x-stackQL-verb": "delete",
   "x-numReqParams": 2,
   "parameters": [
    {
     "name": "account_id",
     "required": true,
     "description": "(uuid) Databricks account ID of any type. For non-E2 account types, get your account ID from the ",
     "in": "path"
    },
    {
     "name": "vpc_endpoint_id",
     "required": true,
     "description": "(uuid) Databricks VPC endpoint ID.",
     "in": "path"
    }
   ],
   "responses": {
    "200": {
     "description": "The VPC endpoint configuration was successfully deleted.",
     "content": {
      "application/json": {
       "schema": {
        "type": "object",
        "description": "example: {}",
        "properties": {}
       }
      }
     }
    },
    "401": {
     "description": "The request does not have valid authentication credentials for the operation."
    },
    "404": {
     "description": "Operation was performed on a resource that does not exist."
    },
    "409": {
     "description": "Request was rejected due a conflict with an existing resource."
    },
    "500": {
     "description": "Internal error."
    }
   },
   "description": "Deletes a VPC endpoint configuration, which represents an"
  }
 }
}